doc/nrf/glossary.rst

```diff
@@ -0,0 +1,431 @@
+
+NFC-A Listen Mode
+   The initial mode of an NFC Forum Device when it does not generate a carrier.
+   The device listens for the remote field of another device.
+   See Near Field Communication (NFC).
+
+Noise Factor (NF)
+   The relation of the Signal-to-Noise Ratio (SNR) in the device input to the SNR in the device output.
+
+Non-volatile Memory Controller (NVMC)
+   A controller used for writing and erasing the internal flash memory and the User Information Configuration Registers (UICR).
+
+nRF Connect for Cloud
+   Nordic Semiconductor's platform for connecting IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor chips, and more.
+   It includes a public REST API that can be used for building IoT solutions.
+   See `nRF Connect for Cloud`_.
+
+nRF repository
+   An |NCS| repository that does not have an externally maintained, open-source upstream.
+   It is exclusive to Nordic development.
+
+nRF Secure Immutable Bootloader (NSIB)
+   A bootloader created and maintained by Nordic Semiconductor that is built on Chain of Trust architecture.
+
+OpenAMP
+   A framework that provides software components that enable the development of software applications for Asymmetric Multiprocessing (AMP) systems.
+   See `OpenAMP`_.
+
+OpenThread
+   A portable and flexible :ref:`open-source implementation <OpenThread.io>` of the Thread networking protocol.
+
+OpenThread Border Router (OTBR)
+   A router that connects a Thread network to other IP-based networks, like Wi-Fi or Ethernet.
+   A Thread network requires a Border Router to connect to other networks.
+
+Operating System (OS)
+   A set of functions and data structures that manages system resources, hardware components, and the execution of programs and processes.
+   It is usually composed of a kernel, a scheduler, a file system, a memory manager, and other components.
+
+OSS repository
+   An |NCS| repository that tracks an upstream Open Source Software counterpart that is externally maintained.
+
+Over-the-Air (OTA)
+   Any type of wireless transmission.
+
+Patch
+   See Commit.
+
+Power Amplifier (PA)
+   A device used to increase the transmit power level of a radio signal.
+
+Power Saving Mode (PSM)
+   A feature introduced in 3GPP Release 12 to improve battery life of IoT (Internet of Things) devices by minimizing energy consumption.
+   The device stays dormant during the PSM window.
+
+Printed Circuit Board (PCB)
+   A board that connects electronic components.
+
+Protocol Data Unit (PDU)
+   Information transferred as a single unit between peer entities of a computer network and containing control and address information or data.
+   PDU mode is one of the two ways of sending and receiving SMS messages.
+
+Provisioning
+   * In a thread mesh network, the process of associating a device to the appropriate service, and performing any application or vendor-specific configuration.
```

Comment:
There is a thread mesh network and there is a Bluetooth mesh network.
Provisioning is used by the Bluetooth mesh to add new devices to a mesh network (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/reference/bluetooth/mesh/provisioning.html#bluetooth-mesh-provisioning)

Thread mesh uses comissioning to add devices to a thread network.
See more details here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thread_commissioning.html

Reply:
In my last commit I clarified in the context of thread what commissioning and provisioning is.
@mia-ko could you please check the new definitions?

Reply:
@mia-ko I missed the first question, sorry. At the moment there is no specific rule on what should be added here.